autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -44,6 +44,16 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
     return meeting.meetingDays.reduce((acc, curr, idx) => (curr ? acc + DAYS_OF_WEEK[idx] : acc), '');
   };
 
+  const getMeetingTimeText = (mtg: Meeting): string => {
+    if (mtg.startTimeHours === 0) {
+      return 'ONLINE';
+    }
```

Comment:
My only problem with this is that checking if the start time is 0 includes all meetings with null start times, which aren't exclusively online classes (for example, I'm taking JAPN 491 next semester, and it has a null meeting time since it's a research class but isn't online, despite showing as online on our website). Most research classes are going to be this way, and there are probably other exceptions. I believe we should check if the section for meeting times of 0 are online—if so, print "ONLINE", and if not, just don't print anything (if we want, we could change it to another message but I think leaving it empty should be fine).

Reply:
Good point, will add